### ISSUE TITLE
limits on number of `TTree` branches

### ISSUE THREAD

**Issue by c-dilks** [drewkenjo/j2root#2]
There seems to be some limit on the number of `TTree` branches that `j2root` can handle (unless this is ROOT's problem). I'm not able to create and fill more than about 200 branches. The test script
```groovy
import org.jlab.jroot.ROOTFile
def root_file = new ROOTFile('test.root')

// settings
//def num_branches = 200 // works
def num_branches = 220 // fails
def data_type = 'F'

def branch_list = (1..num_branches).collect{ 'a'+it+'/'+data_type }
println branch_list
def tree = root_file.makeTree('t', 't', branch_list.join(':'))
1.times{
  val_list = (1..num_branches).collect{1}
  println val_list
  tree.fill(*val_list)
}
tree.write()
root_file.close()
```
returns the error:
```
An irrecoverable stack overflow has occurred.
Please check if any of your loaded .so files has enabled executable stack (see man page execstack(8))
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fc657618941, pid=99778, tid=99810
#
# JRE version: OpenJDK Runtime Environment (11.0.2+9) (build 11.0.2+9)
# Java VM: OpenJDK 64-Bit Server VM (11.0.2+9, mixed mode, tiered, compressed oops, serial gc, linux-amd64)
# Problematic frame:
# C  [libc.so.6+0x154941]  __memmove_ssse3+0x2851
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java a
gain
#
# An error report file with more information is saved as:
# ./hs_err_pid99778.log
```
(let me know if you need `hs_err_pid99778.log`)

I'm not sure how hard of a limit this is... most often 200 branches works fine, but 201 does not (but sometimes 201 works); I recommend adding a warning if a user tries to define a tree with 200 or more branches (or maybe a lower threshold to be safe).

**Comment by c-dilks** [comment]
```
module list
```

```
Currently Loaded Modulefiles:
  1) gcc/9.2.0            5) rcdb/0.06.00         9) clas12root/1.7.8.c  13) ced/1.4.74          17) cmake/3.18.1
  2) python/3.4.3         6) jaw/2.1             10) jdk/11.0.2          14) maven/3.8.5
  3) groovy/4.0.3         7) root/6.24.06        11) coatjava/8.2.2      15) mcgen/2.14
  4) ccdb/1.07.00         8) qadb/1.2.0          12) gemc/4.4.2          16) clas12/pro
```

**Comment by drewkenjo** [comment]
Resolved with the fix df776fbdcd2ecdc8374a0c679cd9a167cdcefa93
